tests/conftest.py

```diff
@@ -20,7 +20,8 @@
 
 skipif_yask = pytest.mark.skipif(configuration['backend'] == 'yask',
                                  reason="YASK testing is currently restricted")
-
+skipif_ops = pytest.mark.skipif(configuration['backend'] == 'ops',
```

Comment:
May I suggest to add something like ``skipif_noncore`` so that whenever you would have both "skipif_yask" and "skipif_ops", you can actually use ``skipif_noncore``? because otherwise this doesn't scale, think about when we start having 5-6 backends...

it'd actually be **fantastic** if you could come up with something that allows one to write
``@skipif('yask', 'ops')`` or maybe ``@skip_backend('yask', 'ops')``

Reply:
That sounds cool indeed. Let me try it!

Reply:
the more I think about it, the more it feels like that this:
``@skip_backend('yask', 'ops')``
would be ideal

Reply:
Ok, I was trying to reach an OK from Travis so I could make this mod.